inputfiles/addedTypes.json

```diff
@@ -400,5 +400,40 @@
         "interface": "Element",
         "name": "getElementsByTagName",
         "signatures": ["getElementsByTagName(tagname: \"picture\"): NodeListOf<HTMLPictureElement>"]
+    },
+    {
+        "kind": "interface",
+        "name": "ParentNode",
+        "flavor": "DOM",
+        "properties": [
+            {
+                "name": "children",
+                "type": "HTMLCollection"
+            },
+            {
+                "name": "firstElementChild ",
+                "type": "Element"
+            },
+            {
+                "name": "lastElementChild ",
+                "type": "Element"
+            },
+            {
+                "name": "childElementCount",
+                "type": "number"
+            }
+        ]
+    },
+    {
+        "kind": "property",
```

Comment:
it should be added to the extends list. i do not know how to do that though :) @zhengbli what is the right way to do this?